app/views/site/copyright.html.erb

```diff
@@ -62,35 +61,88 @@
+                                   "https://www.openstreetmap.org/"),
+              :cc_by_sa_link => link_to(t(".legal_babble.contributors_osm_cc_by_sa"),
+                                        t(".legal_babble.contributors_osm_cc_by_sa_url")) %></li>
+    <li><%= t ".legal_babble.contributors_newberry_ahcb_html",
+              :newberry_link => link_to(t(".legal_babble.contributors_newberry"),
+                                        "https://www.newberry.org/"),
+              :ahcb_link => link_to(tag.cite(t(".legal_babble.contributors_newberry_ahcb")),
+                                    "https://publications.newberry.org/ahcb/") %></li>
+    <li><%= t ".legal_babble.contributors_anr_communes_html",
+              :anr_communes_link => link_to(t(".legal_babble.contributors_anr_communes_project"),
+                                            "https://anrcommunes.hypotheses.org/"),
+              :campop_link => link_to(t(".legal_babble.contributors_anr_communes_campop"),
+                                      t(".legal_babble.contributors_anr_communes_campop_url")) %></li>
+    <li><%= t ".legal_babble.contributors_more_sources_html",
+              :sources_link => link_to(t(".legal_babble.contributors_more_sources_page"),
+                                       t(".legal_babble.contributors_more_sources_page_url")) %></li>
+  </ul>
+  <p><%= t ".legal_babble.contributors_2_ohm_html" %></p>
+  <ul>
+    <li><%= t ".legal_babble.contributors_natural_earth_html",
+              :natural_earth_link => link_to(tag.cite(t(".legal_babble.contributors_natural_earth")),
+                                             "https://www.naturalearthdata.com/") %></li>
+    <li><%= t ".legal_babble.contributors_globcover_html",
+              :globcover_link => link_to(tag.cite(t(".legal_babble.contributors_globcover")),
+                                         "https://due.esrin.esa.int/page_globcover.php") %></li>
+    <li><%= t ".legal_babble.contributors_gmted_html",
+              :gmted_link => link_to(tag.cite(t(".legal_babble.contributors_gmted")),
+                                     "https://pubs.usgs.gov/of/2011/1073/") %></li>
   </ul>
-  <p><%= t ".legal_babble.contributors_footer_1_html", :locale => @locale %></p>
-  <p><%= t ".legal_babble.contributors_footer_2_html", :locale => @locale %></p>
+  <p><%= t ".legal_babble.contributors_footer_2_html" %></p>
 
     <h3><%= t ".legal_babble.infringement_title_html" %></h3>
     <p><%= t ".legal_babble.infringement_1_html" %></p>
+    <p>
+      <%= t ".legal_babble.infringement_2_ohm_html",
+            # Leave the subject untranslated so we know it’s urgent.
+            :contact_team_link => mail_to(Settings.support_email,
+                                          t(".legal_babble.infringement_2_ohm_contact_team"),
+                                          :subject => "TAKEDOWN REQUEST"),
+            :subject => tag.q("TAKEDOWN REQUEST", :lang => "en", :dir => "ltr") %>
```

Comment:
If we end up configuring a legal-specific e-mail address for OpenHistoricalMap/issues#298, we’ll want to link to it here instead of the general team e-mail address.